src/util/routerinfo.cc

```diff
@@ -74,13 +87,99 @@ bool RouterInfoCommand::Impl(
       return false;
     }
 
+  if (create)  // Create new router info + keys
+    {
+      // Sanity checks
+      if (inputs.size() > 1)
+        {
+          LOG(error) << "routerinfo: Too many arguments";
+          PrintUsage(cmd_name);
+          return false;
+        }
+      auto filename = inputs.empty() ? std::string() : inputs.at(0);
+      if (filename == "-")
+        {
+          // Need to output 2 files : RI + key
+          LOG(error)
+              << "routerinfo: output to console is not supported for creation";
+          return false;
+        }
+      auto host = vm["host"].as<std::string>();
+      auto port = vm["port"].defaulted()
+                      ? kovri::core::RandInRange32(9111, 30777)
```

Comment:
Since we're using this port range in multiple places, perhaps we should create a global definition?